### PR TITLE
Lesson 14 TestTicket777 - add new test playListTests_createPlayList_s…

### PR DESCRIPTION
**Pull request by RustamKhudoyarov**
Please do code review

### CODE REVIEW

File: src/test/java/pageObjectsTests/PlayListTests.java [hunk 1]
```diff
@@ -9,14 +9,29 @@
...
+        String playlistId = homePage.createPlaylist(playlistName);
+
+        Assert.assertTrue(homePage.checkNewPlayListExist(playlistId,playlistName));
+
+
 
+    }
     @Test
     public void playListTests_createPlayList_successfulNewPlayList(){
-        // use external lib faker - generate names for your objects
```
Comment (mkhudoiarova) [inline src/test/java/pageObjectsTests/PlayListTests.java:15]: Why do you delete these rows?

File: src/test/java/pageObjectsTests/PlayListTests.java [hunk 2]
```diff
@@ -9,14 +9,29 @@
...
+        Faker faker = new Faker();
+        String playlistName = faker.funnyName().name();
+
+        LoginPage loginPage =new LoginPage(driver);
+        loginPage.openPage();
+
+        HomePage homePage = loginPage.loginToApp(userName,password);
+        String playlistId = homePage.createPlaylist(playlistName);
+
+        Assert.assertTrue(homePage.checkNewPlayListExist(playlistId,playlistName));
```
Comment (mkhudoiarova) [inline src/test/java/pageObjectsTests/PlayListTests.java:24]: I believe, you should use "assertEqual" (instead assertTrue)

**Review by mkhudoiarova — CHANGES_REQUESTED** [review]

File: src/test/java/pageObjectsTests/PlayListTests.java (hunk 1 above)
Comment (RustamKhudoyarov) [reply src/test/java/pageObjectsTests/PlayListTests.java:15]: This in the ticket -777. AC

File: src/test/java/pageObjectsTests/PlayListTests.java (hunk 2 above)
Comment (RustamKhudoyarov) [reply src/test/java/pageObjectsTests/PlayListTests.java]: I agree. I changed method on the assertEquals.

**Review by mkhudoiarova — APPROVED** [review]

**Comment by mkhudoiarova** [comment]
all correct

**Comment by mkhudoiarova** [comment]
reviewed

**Pull request merged**